### PR TITLE
initial announcements

### PR DESCRIPTION
**Pull request by Odyssey346**
This is an initial PR for adding announcements to segfautils.

I might switch from ``gorilla/feeds`` to an actual JSON library instead.

### CODE REVIEW

**Comment by Devnol** [comment]
how do i remove gulag exactly

File: static/announcements.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Segfautils form implementation example</title>
```
Comment (akisblack) [inline static/announcements.html:7]: update the title

**Comment by Odyssey346** [comment]
> how do i review gulag exactly

just get an idea of the code really

**Review by akisblack — CHANGES_REQUESTED** [review]
mostly looks good, i would like the library change though since it will allow blog post linking. i understand that it wont be easy on your side though

File: static/announcements.html (hunk 1 above)
Comment (Odyssey346) [reply static/announcements.html:7]: crap

**Review by Midou36O — APPROVED** [review]
LGTM

**Review by akisblack — APPROVED** [review]

**Comment by Odyssey346** [comment]
> since it will allow blog post linking. 
I never said this would be a "thing". I meant that I'd just get more freedom to do whatever I want with the JSON, unlike ``gorilla/feeds``, which doesn't give you much to work with since it's oriented to RSS feeds (which I totally understand)

If you'd like, we can merge the PR when I switch to a JSON library.

**Comment by akisblack** [comment]
merge it now, we can switch, if we ever want, later

**Comment by Odyssey346** [comment]
Hmm, @Midou36O what's your opinion? Should we hold the PR until I switch the library for handling JSON from one that's oriented towards RSS feeds to an actual JSON library, or should we merge now and change library later?

**Comment by Midou36O** [comment]
i'd rather change the to an actual JSON library before merging.

**Comment by Odyssey346** [comment]
Got it. I will fix this tomorrow, most likely.

**Comment by Odyssey346** [comment]
Switched - re-requesting review

**Review by akisblack — APPROVED** [review]

**Review by Midou36O — APPROVED** [review]
LGTM²

**Pull request merged**